src/plugins/data_view_editor/public/plugin.tsx

```diff
@@ -51,6 +52,7 @@ export class DataViewEditorPlugin
             overlays,
             dataViews,
             searchClient: data.search.search,
+            dataViewEditorService: new DataViewEditorService(http, dataViews),
```

Comment:
this should be made lazy somehow - more than double the bundle size is concerning for a refactoring

Reply:
You're right, although I'm going to put this off for now. I reduce the limit.yml entry so its increased to 15k. The page load bundle is 4k larger.

Reply:
update - page load bundle size is 150 bytes larger